spec/units/tab_completion/matchers/tag_matchers_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'spec_helper'
+require 'gitsh/tab_completion/matchers/tag_matcher'
+
+describe Gitsh::TabCompletion::Matchers::TagMatcher do
+  describe '#match?' do
+    it 'always returns true' do
+      matcher = described_class.new(double(:env))
+
+      expect(matcher.match?('foo')).to be_truthy
```

Comment:
Does

```ruby
expect(matcher).to match('foo')
```

work?

Reply:
Sadly not. I think it would if the method was named differently, but there is already a [`match` matcher][1] in RSpec that takes precedence over the dynamic predicate matchers.

[1]: https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/match-matcher